### PR TITLE
alpha workflow: added step to delete alpha to avoid outdate release

### PR DESCRIPTION
**Pull request by nathanmartinszup**
Signed-off-by: nathanmartinszup <nathan.martins@zup.com.br>

**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: .github/workflows/alpha.yml [hunk 1]
```diff
@@ -77,6 +77,14 @@ jobs:
           COSIGN_PWD: ${{ secrets.COSIGN_PWD }}
           GORELEASER_CURRENT_TAG: v0.0.0-alpha
 
+      - name: Delete outdate release
```
Comment (matheusalcantarazup) [inline .github/workflows/alpha.yml:80]: I notice that the [alpha](https://github.com/ZupIT/horusec/tree/alpha) tag is not being synchronized with main, why do wee need delete the release? Synchronize the alpha tag with main and update the release it is not enough?

File: .github/workflows/alpha.yml [hunk 2]
```diff
@@ -77,6 +77,14 @@ jobs:
...
+        uses: dev-drprasad/delete-tag-and-release@v0.1.2
```
Comment (wiliansilvazup) [inline .github/workflows/alpha.yml:81]: I'm afraid of using this action. Would it be too much work to do manually?

Comment (nathanmartinszup) [reply .github/workflows/alpha.yml:81]: It would be more work and we would have the same result.

File: .github/workflows/alpha.yml (hunk 1 above)
Comment (nathanmartinszup) [reply .github/workflows/alpha.yml:80]: It would be more work and we would have the same result.

**Review by matheusalcantarazup — APPROVED** [review]

**Pull request merged**